var/spack/repos/builtin/packages/py-blis/package.py

```diff
@@ -18,3 +18,7 @@ class PyBlis(PythonPackage):
 
     depends_on("py-setuptools", type="build")
     depends_on("py-numpy@1.15:", type=("build", "run"))
+
+    def setup_build_environment(self, env):
+        env.set("BLIS_COMPILER", self.compiler.cc)
```

Comment:
Is this the compiler used at build time or the compiler used at run time? If the former, you'll want to use:
```suggestion
        env.set("BLIS_COMPILER", spack_cc)
```
so that it uses Spack's compiler wrappers.